pkg/agent/clientset.go

```diff
@@ -110,7 +110,7 @@ type ClientSetConfig struct {
 	AgentID                 string
 	SyncInterval            time.Duration
 	ProbeInterval           time.Duration
-	DialOption              grpc.DialOption
+	DialOption              []grpc.DialOption
```

Comment:
And here? Public so requires a bit more thought.

Reply:
Should be no problem renaming this since the agent is a standalone binary and is not imported anywhere.